docs/repositories/issues.md

```diff
@@ -98,7 +98,13 @@ To restore an ignored issue, select **Unignore issue** from the options menu:
 
 !!! info "This section applies to GitHub repositories only"
 
-If Codacy detects code patterns that have suggested fixes, a **Fix issues** button appears above the issue list. Click this button to open a pull request including the suggested fixes and targeting the selected branch.
+If Codacy detects code patterns with suggested fixes, a **Fix issues** button appears above the issue list. Click this button to open a modal with a patch that addresses all resolvable issues. Follow the instructions in the modal to apply the patch:
```

Comment:
I feel that "Click this button to open a modal" would be the first step of the procedure... 😉 
It might not be so easy to solve, as the button is what indicates that there are available fixes...

Leaving a suggestion below.

```suggestion
If Codacy detects code patterns with suggested fixes, a **Fix issues** button appears above the issue list.

![Fix issues button](images/issues-fix-issues-button.png)

In this case, Codacy generates a patch that enables you to solve all resolvable issues. To apply this patch to your local branch, do the following:

1.  Click the button **Fix issues** to open a modal with a patch that addresses all resolvable issues.
```